ui/src/app/subnets/views/Subnets.tsx

```diff
@@ -8,33 +8,33 @@ import SubnetDetails from "app/subnets/views/SubnetDetails";
 import SubnetsList from "app/subnets/views/SubnetsList";
 import VLANDetails from "app/subnets/views/VLANDetails";
 
-const Routes = (): JSX.Element => {
+const Subnets = (): JSX.Element => {
   return (
     <Switch>
-      <Route exact path={subnetsURLs.index} render={SubnetsList} />
```

Comment:
Also these routes needed to be changed to render the component without passing props through the route.